src/databases/models/User.js

```diff
@@ -62,7 +62,10 @@ module.exports = (sequelize, DataTypes) => {
       profile_pic_public_id: {type: DataTypes.STRING, allowNull: true},
       status: {type: DataTypes.BOOLEAN, allowNull: false},
       bio: {type: DataTypes.TEXT, allowNull: true},
-      blocked_by_admin: {type: DataTypes.BOOLEAN, allowNull: true}
+      blocked_by_admin: {type: DataTypes.BOOLEAN, allowNull: true},
+      is_anonymous: {type: DataTypes.BOOLEAN, allowNull: false},
+      combined_user_score: {type: DataTypes.DOUBLE, allowNull: false},
+      karma_score: {type: DataTypes.DOUBLE, allowNull: false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `is_anonymous`, `combined_user_score`, and `karma_score` are set to not allow null values (`allowNull: false`). However, for existing users in the database, these fields will be null after the migration. This could cause issues when trying to retrieve or update these users. Consider setting a default value or allowing null.

```diff
-      is_anonymous: {type: DataTypes.BOOLEAN, allowNull: false},
-      combined_user_score: {type: DataTypes.DOUBLE, allowNull: false},
-      karma_score: {type: DataTypes.DOUBLE, allowNull: false}
+      is_anonymous: {type: DataTypes.BOOLEAN, allowNull: false, defaultValue: false},
+      combined_user_score: {type: DataTypes.DOUBLE, allowNull: true},
+      karma_score: {type: DataTypes.DOUBLE, allowNull: true}
```
This change sets a default value of `false` for `is_anonymous` and allows `null` for `combined_user_score` and `karma_score`. You may want to adjust these defaults based on your application's needs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->